test/Testing/LeanCode.IntegrationTestHelpers.Tests/ConfigurationOverridesTests.cs

```diff
@@ -0,0 +1,183 @@
+using Microsoft.Extensions.Configuration;
+using Xunit;
+
+namespace LeanCode.IntegrationTestHelpers.Tests;
+
+public class ConfigurationOverridesTests
+{
+    [Fact]
+    public void Empty_ConfigurationOverrides_Results_In_Empty_Configuration_test()
+    {
+        var configuration = new ConfigurationBuilder().Add(new ConfigurationOverrides()).Build();
+
+        Assert.Empty(configuration.AsEnumerable());
+    }
+
+    [Fact]
+    public void ConfigurationOverrides_single_Add_method_test()
```

Comment:
This name (and the others below) follows a totally different convention to the first test. We need to be consistent and name it, for example, `Adding_single_value_results_in_single_value_in_configration`.

Reply:
Also, you don't need to specify `ConfigurationOverrides` in each class name - that is given by the class name.